### DISCUSSION TITLE
How to set `--release` as default for `pip install -e .`

### DISCUSSION THREAD

**Discussion by sun-rs** [PyO3/maturin#1999] [Q&A]
It seems that maturin don't use `--release` mode as default, when I run maturin build or develop it returns output logs like `[unoptimized + debuginfo]`, it is ok to add `--release` manaully. But when using `pip install .` or `pip install -e .`, how to set `--release` flag?

**Comment by messense** [comment]
You don't, it's release by default in PEP 517 build.

**Reply by sun-rs** [reply]
Sorry， I'm not quite understand, when I put `maturin build` it fininsh with `Finished dev profile [unoptimized + debuginfo] target(s) in 13.28s`, only if you put `maturin build --release` it would finish with `Finished release profile [optimized] target(s) in 20.99s`
